### DISCUSSION TITLE
Error Signing In with Supabase Custom Claims in React Native Expo App

### DISCUSSION THREAD

**Discussion by laneeey** [supabase/supabase#27807] [Questions] [answered]
I am encountering an error when attempting to sign in to my React Native app using Supabase after setting up custom claims (everything was working before its specifically related to the new claim). The error message is: 

`Error signing in: Error running hook URI: pg-functions://postgres/public/custom_access_token_hook`

### Steps to Reproduce:

Followed the [Supabase custom claims and role-based access control (RBAC) documentation](https://supabase.com/docs/guides/database/postgres/custom-claims-and-role-based-access-control-rbac?queryGroups=language&language=plpgsql).

1. Created the necessary tables and the custom_access_token_hook function as described.
2. Ran the grant commands provided in the documentation.
3. Enabled the hook in the Supabase project settings.
4. Created the authorize function as given in the docs.
5. Attempted to sign in using the Supabase createClient function in my React Native app.
6. Expected Behavior:
7. Successful sign-in to the React Native app using Supabase with custom claims.

### Actual Behavior:
Sign-in fails with the error message: Error signing in: Error running hook URI: pg-functions://postgres/public/custom_access_token_hook.

### Additional Information:
I am not running Supabase locally; I am connecting to Supabase using the createClient function.
I suspect the issue might be related to permissions or Row-Level Security (RLS) but could be wrong?

### Questions:

1. Is this error occurring because the hook is attempting to use a local Supabase instance?
2. Is it more likely that the issue is related to permissions or RLS configuration?
3. Any insights or solutions to help resolve this error would be greatly appreciated.

Thank you!

**Comment by GaryAustin1** [comment, marked as answer]
Look in the Postgres logs for errors. There are dozens of different things it could be including grants, RLS, bad naming, etc.

**Reply by laneeey** [reply]
Thanks for your help!

It turns out the issue was that the role column doesn't exist, `column "role" does not exist`.

I'm relatively new to Supabase & Postgres now I know where to look for DB-related logs in the future.
